api/helm.go

```diff
@@ -153,6 +154,24 @@ func ListDeployments(c *gin.Context) {
 		}
 	}
 
+	// Create WhiteList set
+	securityClientSet := getSecurityClient(c)
+	if securityClientSet == nil {
+		log.Errorf("can't get security clientset: %s", err)
+	}
+	releaseWhitelist := make(map[string]bool)
+	if securityClientSet != nil {
```

Comment:
This should be an else if block I think.

Reply:
true & fixed